osmaxx-py/excerptconverter/dummyexcerptconverter/dummy_excerpt_converter.py

```diff
@@ -120,7 +126,7 @@ def execute_task(extraction_order_id, supported_export_formats, execution_config
             # now set the new state (if all files have been processed) and inform the user about the state
             converter_helper.file_conversion_finished()
         except:
-            # TODO: log stack trace
             message_text = _('The Dummy conversion of extraction order "%s" failed.') % extraction_order.id
             converter_helper.inform_user(messages.ERROR, message_text, email=False)
+            logger.exception()
```

Comment:
[Python docs](https://docs.python.org/3.4/library/logging.html#logging.Logger.exception) state:
```
Logger.exception(msg, *args, **kwargs)
Logs a message with level ERROR on this logger. The arguments are interpreted as for debug().
Exception info is added to the logging message. This method should only be called from an exception handler.
```

Doesn't that mean, I don't have to add any information, because it will be added to the log-message anyway?